.github/workflows/test.yml

```
@@ -0,0 +1,20 @@
+---
+name: tests
+on: push
```

Should we test on PR, too?
```suggestion
on: [push, pull_request]
```

---

👍